cromulent/vocab.py

```diff
@@ -50,12 +51,16 @@ def register_instance(name, data):
 		t = parent("http://vocab.getty.edu/aat/%s" % id)
 	t._label = label
 	instances[name] = t
+	return t
 
 # Meta meta
 ext_classes = {
 	"LocalNumber": {"parent":Identifier, "id":"300404621", "label": "Local Number"},	
 	"AccessionNumber": {"parent":Identifier, "id":"300312355", "label": "Accession Number"},
 	"LotNumber": {"parent": Identifier, "id": "300404628", "label": "Lot Number"},
+	"IsbnIdentifier": {"parent": Identifier, "id": "300417443", "label": "ISBN Identifier"},
```

Comment:
Is your preference to use this capitalization for an initialism (compared to `ISBNIdentifier`)? Or is it a python style-thing, perhaps?

Reply:
Vague personal preference, in that DOIIdentifier is easier to typo than DoiIdentifier (and similar)

Reply:
Harder to type?